tpm_futurepcr/PcrBank.py

```diff
@@ -0,0 +1,84 @@
+import hashlib
+import os
+import subprocess as sp
+from typing import Optional
+import re
+import logging
+
+from .PcrRegister import PcrRegister
```

Comment:
Still not sure why you insist on CamelCase for module names (Java style is not Python style), but alright...

Reply:
my bad, let me correct that: That is what I am used to at work and... yeah :-/